src/doc/not_found.md

```diff
@@ -84,7 +84,12 @@ function on_submit(event) {
     if (form['from'].value === 'duckduckgo') {
         document.location.href = form.action + '?q=' + encodeURIComponent(q + ' site:doc.rust-lang.org');
     } else if (form['from'].value === 'library') {
-        document.location.href = 'std/index.html?search=' + encodeURIComponent(q);
+        var path = window.location.pathname.split('/');
+        var version = "";
+        if (path.length > 0 && path[0] === "nightly" || path[0] === "beta") {
```

Comment:
Won't this also miss situations where it's on a version-specific doc bundle too, like https://doc.rust-lang.org/1.28.0/ ?

Also, i'm worried it will try to index `path[0]` to check for `"beta"` when it's empty, because of how the order of operations works out...

Reply:
It's a tricky issue. Honestly, I'm not sure what's the best course here...